tests/cloner_test.go

```diff
@@ -105,7 +107,9 @@ var _ = Describe("all clone tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Deleting verifier pod")
-			err = f.K8sClient.CoreV1().Pods(f.Namespace.Name).Delete(context.TODO(), utils.VerifierPodName, metav1.DeleteOptions{})
+			err = f.K8sClient.CoreV1().Pods(f.Namespace.Name).Delete(context.TODO(), utils.VerifierPodName, metav1.DeleteOptions{
+				GracePeriodSeconds: &zero,
```

Comment:
Why not add DeletePodImmediately to utils in tests/utils/pod.go, then It can also be used here:
tests/framework/pvc.go:107 in createConsumerPod and maybe some other places in the future

Reply:
Yeah I thought about that. At this point I was trying to get NFS stable and happy. If that is true, then I will probably add a function like that, as I think there are quite a few places that could use this.